code/src/styles.css

```diff
@@ -0,0 +1,209 @@
+body {
+  background-image: url("./assets/bouba-kiki-effect.avif");
+  background-size: cover;
+}
+
+.app-container {
+height: 100vh;
+width: 100vw;
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-items: center;
+overflow: hidden;
+}
+
+.app-container span {
+  color: rgb(88, 80, 80);
+  background-color: white;
+  font-family: Manga;
+  align-items: center;
+  text-align: center;
+  font-size: 40px;
```

Comment:
I did exactly this my self so well - for both of us - maybe try to write margins and paddings in one line when we use more than one of them (Like margin: 10px 5px 5px 10px;) It's just a bit hard to remember wich one do margins where I think.